python/lsst/obs/subaru/rings.py

```diff
@@ -1116,17 +1154,17 @@ def plotRadialProfile(mos, visits, butler=None, showMedians=True, showMeans=Fals
                 filterName = butler.get("raw_md", visit=v, ccd=10).get("FILTER01").lower()
                 if re.search(r"^hsc-", filterName):
                     filterName = filterName[-1:]
-                if filterName == "z" and v in range(904672, 905068+1, 2): # z wasn't available
-                    filterName = "i"                                      # error in header
+                if filterName == "z" and v in range(904672, 905068+1, 2):  # z wasn't available
+                    filterName = "i"                                       # error in header
             else:
                 filterName = "unknown"
     else:
         raise RuntimeError("Please provide at least on visit for me to plot")
 
     if LaTeX:
-        print r"""\begin{longtable}{lllll}
+        print(r"""\begin{longtable}{lllll}
 visit & median flux & exposure time & line type & line colour \\
-\hline"""
+\hline""")
```

Comment:
No: raw strings are raw with all formatting intact, so they have to look like this for latex (as far as I'm aware).